README.md

```diff
@@ -175,9 +176,10 @@ strategies:
       nodeAffinityType:
       - "requiredDuringSchedulingIgnoredDuringExecution"
 ```
+
 ### RemovePodsViolatingNodeTaints
 
-This strategy makes sure that pods violating NoSchedule taints on nodes are removed. For example: there is a pod "podA" with toleration to tolerate a taint ``key=value:NoSchedule`` scheduled and running on the tainted node. If the node's taint is subsequently updated/removed, taint is no longer satisfied by its pods' tolerations and will be evicted. The policy file should look like:
+This strategy makes sure that pods violating NoSchedule taints on nodes are removed. For example there is a pod "podA" with a toleration to tolerate a taint ``key=value:NoSchedule`` scheduled and running on the tainted node. If the node's taint is subsequently updated/removed, taint is no longer satisfied by its pods' tolerations and will be evicted. The policy file should look like this.
```

Comment:
`:` is still more suitable here
can you break the line into more lines?

Reply:
Fixed